exporter/awsemfexporter/emf_exporter.go

```diff
@@ -111,11 +110,20 @@ func (emf *emfExporter) pushMetricsData(_ context.Context, md pdata.Metrics) (dr
 
 	rms := md.ResourceMetrics()
 
+	labels := map[string]string{}
 	for i := 0; i < rms.Len(); i++ {
 		rm := rms.At(i)
+		am := rm.Resource().Attributes()
+		if am.Len() > 0 {
+			am.ForEach(func(k string, v pdata.AttributeValue) {
+				labels[k] = v.StringVal()
+			})
+		}
```

Comment:
why can't we directly take the labels from `groupedMetrics.Labels`?

Reply:
The `labels` only serves for the purpose of logging. It should be simple and timely. I moved `translateOTelToGroupedMetric` from the label extraction to another independent iteration. Another concern is since `groupMetrics.Labels` contains extra labels from datapoints, they would blow up the info log.